src/main/java/com/ldtteam/blockui/controls/AbstractTextElement.java

```diff
@@ -267,11 +267,6 @@ else if (textBlock instanceof final FormattedToggleableCharSequence toggleable)
         }
     }
 
-    protected int getTextRenderingColor(final double mx, final double my)
```

Comment:
not used anywhere else?

Reply:
yep, and provides logically old/unstable data